plugin.video.southpark_unofficial/changelog.txt

```diff
@@ -46,4 +46,6 @@
 0.4.2
 - Fixed DE language (thanks to JinJin @kodi forums)
 0.4.3
-- Added Season 20
\ No newline at end of file
+- Added Season 20
+0.4.4
+- Fixed EN/DE/ES streams.
```

Comment:
It makes more sense to show newest changes at the top of the file as users see those first.